haproxy/defaults/main.yml

```diff
@@ -0,0 +1,7 @@
+haproxy-host-name-1: ""
+haproxy-host-name-2: ""
+haproxy-host-name-3: ""
+
+haproxy-host-ip-1: ""
+haproxy-host-ip-2: ""
+haproxy-host-ip-3: ""
```

Comment:
These should be lists and then loop in the template so its the most flexible

Reply:
Created list and loop